rust/crd/src/authentication/mod.rs

```diff
@@ -52,11 +52,7 @@ pub enum Error {
 #[derive(Clone, Deserialize, Debug, Eq, JsonSchema, PartialEq, Serialize)]
 #[serde(rename_all = "camelCase")]
 pub struct DruidAuthentication {
-    /// The AuthenticationClass <https://docs.stackable.tech/home/nightly/concepts/authenticationclass.html> to use.
-    ///
-    /// ## TLS provider
-    /// Please note that the SecretClass used to authenticate users needs to be the same
-    /// as the SecretClass used for internal communication.
+    /// The name of an [AuthenticationClass](DOCS_BASE_URL_PLACEHOLDER/concepts/authentication) object.
```

Comment:
Why remove the documentation of the tls provider restriction? ;)

Reply:
https://github.com/stackabletech/druid-operator/pull/494/commits/8adfa6050d12e6de56e9c487f1360cd52bb6fca9